labs/compreview-todo/todo-frontend/Containerfile

```diff
@@ -0,0 +1,30 @@
+FROM registry.access.redhat.com/ubi8/nodejs-16 AS appbuild
```

Comment:
Some comment about switching to an internal image.

Reply:
Fixing this one...